examples/kube-container-collection/main.go

```diff
@@ -105,7 +105,7 @@ func main() {
 		*kubeconfig = os.Getenv("KUBECONFIG")
 	}
 
-	if *node == "" && os.Getenv("NODE_NAME") != "" {
+	if *node == "" && os.("NODE_NAME") != "" {
```

Comment:
I don't understand the syntax `os.("NODE_NAME")`

Reply:
Neither do I :D. Fixed!